<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>choose_model</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
border-image: url(:/image1/resource/classroom.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>1280</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
        border-image: url(:/image1/resource/message.png);
    }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{letter-spacing:5px;}</string>
     </property>
     <property name="text">
      <string>Narrator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_content">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>681</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>15</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>140</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image2/resource/qload.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>140</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image2/resource/title.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>140</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image2/resource/screen.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>140</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image2/resource/qsave.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>70</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_input">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>601</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_option1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
    image: url(:/image1/resource/button.png);
    background-color:transparent;
    border:None;
    }
    #pushButton:hover{
        image: url(:/image1/resource/button2.png);
    }
    #pushButton:pressed{
        image:url(:/image1/resource/button3.png);
    }</string>
    </property>
    <property name="text">
     <string>Option1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_option2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
    image: url(:/image1/resource/button.png);
    background-color:transparent;
    border:None;
    }
    #pushButton_2:hover{
        image: url(:/image1/resource/button2.png);
    }
    #pushButton_2:pressed{
        image:url(:/image1/resource/button3.png);
    }</string>
    </property>
    <property name="text">
     <string>Option2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_option3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
    image: url(:/image1/resource/button.png);
    background-color:transparent;
    border:None;
    }
    #pushButton_3:hover{
        image: url(:/image1/resource/button2.png);
    }
    #pushButton_3:pressed{
        image:url(:/image1/resource/button3.png);
    }</string>
    </property>
    <property name="text">
     <string>Option3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/people/resource/girl_smile-removebg-preview.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/people/resource/girl_shy-removebg-preview (2).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_right">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/people/resource/boy_normal_-removebg-preview.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_img_left</zorder>
   <zorder>label_8</zorder>
   <zorder>label_img_right</zorder>
   <zorder>frame</zorder>
   <zorder>pushButton_option1</zorder>
   <zorder>pushButton_option2</zorder>
   <zorder>pushButton_option3</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resource1.qrc"/>
 </resources>
 <connections/>
</ui>
